decidim-admin/app/queries/decidim/admin/freetext_scopes.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    # This query searches scopes by name.
+    class FreetextScopes < Rectify::Query
```

Comment:
Yes, I've been looking at that gem too, and I also reach the same conclusion too :smile: